app/helpers/s3_helper.rb

```diff
@@ -0,0 +1,8 @@
+module S3Helper
```

Comment:
I know you are responding to Mark's comment but I'd rather see this in `S3Util`. I think it is better to use helpers only when sharing instance state. Then the dependencies are clearer.

Reply:
Sounds good, I've moved it to `S3Util`!